### PR TITLE
Add time to map itin summary overlay

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**

Adds a time duration to the itinerary summary overlay! Follows SEPTA style

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/map/itinerary-summary-overlay.tsx [hunk 1]
```diff
@@ -57,7 +70,8 @@ const Card = styled.div`
     }
   }
   div {
-    margin-top: -0px!important;
+    margin-top: -0px !important;
```
Comment (amy-corson-ibigroup) [inline lib/components/map/itinerary-summary-overlay.tsx:73]: I doing think this `-` is doing anything!

File: lib/components/map/itinerary-summary-overlay.tsx [hunk 2]
```diff
@@ -112,7 +126,8 @@ function getUniquePoint(
 
     const selfDistance = distance(point, centerOfLine)
     // maximize distance from all other points while minimizing distance to center of our own line
-    const averageDistance = totalDistance / otherMidpoints.length - selfDistance
+    const averageDistance =
+      1.2 * (totalDistance / otherMidpoints.length) - selfDistance
```
Comment (amy-corson-ibigroup) [inline lib/components/map/itinerary-summary-overlay.tsx:130]: What's this 1.2 do?

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks really good!

File: lib/components/map/itinerary-summary-overlay.tsx (hunk 2 above)
Comment (miles-grant-ibigroup) [reply lib/components/map/itinerary-summary-overlay.tsx:130]: This prioritizes the labels not overlapping slightly above them being in the center of the line. I'm not super happy with this value, so we'll probably tweak it more in the future

File: lib/components/map/itinerary-summary-overlay.tsx (hunk 1 above)
Comment (miles-grant-ibigroup) [reply lib/components/map/itinerary-summary-overlay.tsx:73]: Good catch haah

**Review by josh-willis-arcadis — APPROVED** [review]
LGTM

**Pull request merged**